EssentialsGeoIP/pom.xml

```diff
@@ -18,31 +18,25 @@
         </license>
     </licenses>
 
-    <scm>
-        <connection>scm:git:https://github.com/essentials/Essentials.git</connection>
-        <developerConnection>scm:git:https://github.com/essentials/Essentials.git</developerConnection>
-        <url>https://github.com/essentials/Essentials</url>
-    </scm>
-
     <dependencies>
-        <dependency>
-            <groupId>net.ess3</groupId>
-            <artifactId>EssentialsX</artifactId>
-            <version>${project.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>com.maxmind.geoip</groupId>
-            <artifactId>geoip-api</artifactId>
-            <version>1.2.11</version>
-        </dependency>
+            <dependency>
+                <groupId>net.ess3</groupId>
+                <artifactId>EssentialsX</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.maxmind.geoip</groupId>
+                <artifactId>geoip-api</artifactId>
+                <version>1.2.11</version>
+            </dependency>
     </dependencies>
 
     <build>
         <plugins>
```

Comment:
Why did you refrain from indenting anything in the `build` tag?

Reply:
That wasn't meant to be indented, will fix